docs/_documentations/remote-deploying-codewind.md

```diff
@@ -164,15 +164,27 @@ Your Codewind instance is deployed. Access it by way of the **Access URL** retur
 
 **Note:** When you are installing on Windows, the script runs and then closes the Git Bash window. To retrieve the Access URLs for Codewind, enter the following command:
 
-`$ kubectl get codewind -n codewind`
+`$ kubectl get codewinds -n codewind`
 
 This command returns output in the format of the following example:
 
 ```
-NAME       USERNAME   NAMESPACE   AGE    KEYCLOAK   REGISTRATION   ACCESSURL
-devex001   jane       codewind    119m   devex001   Complete       https://codewind-gatekeeper-devex001.<ingress-domain>
+NAME    USERNAME   NAMESPACE   WORKSPACE      AGE    KEYCLOAK   REGISTRATION   ACCESSURL
+jane1   jane       codewind    kbc3b0x2qins   119m   devex001   Complete       https://codewind-gatekeeper-kbc3b0x2qins.codewind.<ingress-domain>
 ```
 
+For the 0.14.0 release of Codewind, a new `WORKSPACE` field appears in the output. This field is a randomly generated set of characters that are used in both the URL of the Gatekeeper and the name of the pods and resources that you create. 
```

Comment:
The user doesn't create or use the WORKSPACE field other than to use it for identification. Reading this sentence it suggests this value will appear in PROJECT urls which it does not. I think we can just snip off "that you create"

Reply:
Corrected.